packages/@lwc/aria-reflection/tsconfig.json

```diff
@@ -2,7 +2,6 @@
     "extends": "../../../tsconfig.json",
 
     "compilerOptions": {
-        "sourceMap": false,
```

Comment:
At first I was confused by this, but it looks like removing this line from `tsconfig.json` has no effect, either with or without Rollup. I checked, and all our packages have the proper `.map` files in the `dist/`.